Meshtastic/MeshtasticApp.swift

```diff
@@ -26,6 +29,28 @@ struct MeshtasticAppleApp: App {
 		let appState = AppState(
 			router: Router()
 		)
+		// Initialize Datadog
+		// RUM Client Tokens are NOT secret
+		let appID = "79fe92a9-74c9-4c8f-ba63-6308384ecfa9"
+		let clientToken = "pub4427bea20dbdb08a6af68034de22cd3b"
+		let environment = "testflight"
+
+		Datadog.initialize(
+			with: Datadog.Configuration(
+				clientToken: clientToken,
+				env: environment,
+				site: .us5
+			),
+			trackingConsent: .granted
```

Comment:
This is definitely not production-worthy

Reply:
Updated with an opt out user default, matches android